build.gradle

```diff
@@ -60,5 +65,5 @@ run {
 }
 
 wrapper {
-    gradleVersion = '5.0'
+    gradleVersion = '6.5.1'
 }
```

Comment:
@colinanderson I do wonder if we need to specify the gradle version in the script when we already have the wrapper in the repository.  I guess its useful for information though...

Reply:
I think this is only for creating the wrapper so shouldn't be needed if the wrapper files are already there.

Reply:
I think the wrapper task is only needed if you want to create a wrapper and isn't needed if you already have the wrapper.